### PR TITLE
Allow the module to be used on Terraform 0.12

### LINKED ISSUE #9: Modify output names to support Terraform 0.12
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* ***If you are interested in working on this issue or have submitted a pull request, please leave a comment!***

***

### Summary

```
Initializing modules...
Downloading traveloka/waf-owasp-top-10-rules/aws 0.1.1 for waf...
- waf in .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71
There are some problems with the configuration, described below.

The Terraform configuration must be valid before initialization so that
Terraform can determine which modules and providers need to be installed.

Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 6, in output "01_sql_injection_rule_id":
   6: output "01_sql_injection_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 11, in output "02_auth_token_rule_id":
  11: output "02_auth_token_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 16, in output "03_xss_rule_id":
  16: output "03_xss_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 21, in output "04_paths_rule_id":
  21: output "04_paths_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 26, in output "06_php_insecure_rule_id":
  26: output "06_php_insecure_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 31, in output "07_size_restriction_rule_id":
  31: output "07_size_restriction_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 36, in output "08_csrf_rule_id":
  36: output "08_csrf_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.


Error: Invalid output name

  on .terraform/modules/waf/traveloka-terraform-aws-waf-owasp-top-10-rules-8933a71/outputs.tf line 41, in output "09_server_side_include_rule_id":
  41: output "09_server_side_include_rule_id" {

A name must start with a letter and may contain only letters, digits,
underscores, and dashes.
```

### Module Version

0.1.1

### Terraform Version and Terraform AWS Provider Version

```
Terraform v0.12.5
+ provider.aws v2.21.1
```

### Affected Resource(s)

outputs.tf

### Terraform Configuration Files

Not required.

### Expected Behavior

It should work fine

### Actual Behavior

It doesn't.

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

Not required.

### References

None

### PR DESCRIPTION
**Pull request by julianxhokaxhiu**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

***

Fixes #9

***

Release note for [CHANGELOG](https://github.com/traveloka/terraform-aws-waf-owasp-top-10-rules/blob/master/CHANGELOG.md):

```release-note
NOTES:

* This PR allows the module to be used again. Works with both global and regional.

FEATURES:

* NONE

ENHANCEMENTS:

* NONE

BUG FIXES:

* Allow the module to be run again through Terraform
```

***

Output from `terraform plan` command from changes you propose.

```
$ terraform plan
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.waf.aws_waf_byte_match_set.owasp_02_auth_token_string_set[0] will be created
  + resource "aws_waf_byte_match_set" "owasp_02_auth_token_string_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".TJVA95OrM7E2cBab30RMHrHDcEfxjoYZgeFONFh7HgQ"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + data = "authorization"
              + type = "HEADER"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "example-session-id"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + data = "cookie"
              + type = "HEADER"
            }
        }
    }

  # module.waf.aws_waf_byte_match_set.owasp_04_paths_string_set[0] will be created
  + resource "aws_waf_byte_match_set" "owasp_04_paths_string_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "../"
          + text_transformation   = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "../"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "://"
          + text_transformation   = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "://"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "../"
          + text_transformation   = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "../"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "://"
          + text_transformation   = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "://"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
    }

  # module.waf.aws_waf_byte_match_set.owasp_06_php_insecure_qs_string_set[0] will be created
  + resource "aws_waf_byte_match_set" "owasp_06_php_insecure_qs_string_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "_ENV["
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "_SERVER["
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "allow_url_include="
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "auto_append_file="
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "auto_prepend_file="
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "disable_functions="
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "open_basedir="
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "CONTAINS"
          + target_string         = "safe_mode="
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
    }

  # module.waf.aws_waf_byte_match_set.owasp_06_php_insecure_uri_string_set[0] will be created
  + resource "aws_waf_byte_match_set" "owasp_06_php_insecure_uri_string_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = "/"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = "php"
          + text_transformation   = "URL_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
    }

  # module.waf.aws_waf_byte_match_set.owasp_08_csrf_method_string_set[0] will be created
  + resource "aws_waf_byte_match_set" "owasp_08_csrf_method_string_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + byte_match_tuples {
          + positional_constraint = "EXACTLY"
          + target_string         = "post"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "METHOD"
            }
        }
    }

  # module.waf.aws_waf_byte_match_set.owasp_09_server_side_include_string_set[0] will be created
  + resource "aws_waf_byte_match_set" "owasp_09_server_side_include_string_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".backup"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".bak"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".cfg"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".conf"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".config"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".ini"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "URI"
            }
        }
      + byte_match_tuples {
          + positional_constraint = "ENDS_WITH"
          + target_string         = ".log"
          + text_transformation   = "LOWERCASE"

          + field_to_match {
              + type = "URI"
            }
        }
    }

  # module.waf.aws_waf_rule.owasp_01_sql_injection_rule[0] will be created
  + resource "aws_waf_rule" "owasp_01_sql_injection_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "SqlInjectionMatch"
        }
    }

  # module.waf.aws_waf_rule.owasp_02_auth_token_rule[0] will be created
  + resource "aws_waf_rule" "owasp_02_auth_token_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "ByteMatch"
        }
    }

  # module.waf.aws_waf_rule.owasp_03_xss_rule[0] will be created
  + resource "aws_waf_rule" "owasp_03_xss_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "XssMatch"
        }
    }

  # module.waf.aws_waf_rule.owasp_04_paths_rule[0] will be created
  + resource "aws_waf_rule" "owasp_04_paths_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "ByteMatch"
        }
    }

  # module.waf.aws_waf_rule.owasp_06_php_insecure_rule[0] will be created
  + resource "aws_waf_rule" "owasp_06_php_insecure_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "ByteMatch"
        }
      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "ByteMatch"
        }
    }

  # module.waf.aws_waf_rule.owasp_07_size_restriction_rule[0] will be created
  + resource "aws_waf_rule" "owasp_07_size_restriction_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "SizeConstraint"
        }
    }

  # module.waf.aws_waf_rule.owasp_08_csrf_rule[0] will be created
  + resource "aws_waf_rule" "owasp_08_csrf_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "ByteMatch"
        }
      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "SizeConstraint"
        }
    }

  # module.waf.aws_waf_rule.owasp_09_server_side_include_rule[0] will be created
  + resource "aws_waf_rule" "owasp_09_server_side_include_rule" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + predicates {
          + data_id = (known after apply)
          + negated = false
          + type    = "ByteMatch"
        }
    }

  # module.waf.aws_waf_rule_group.owasp_top_10[0] will be created
  + resource "aws_waf_rule_group" "owasp_top_10" {
      + id          = (known after apply)
      + metric_name = (known after apply)
      + name        = (known after apply)

      + activated_rule {
          + priority = 1
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
      + activated_rule {
          + priority = 2
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
      + activated_rule {
          + priority = 3
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
      + activated_rule {
          + priority = 4
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
      + activated_rule {
          + priority = 5
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
      + activated_rule {
          + priority = 6
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
      + activated_rule {
          + priority = 7
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
      + activated_rule {
          + priority = 8
          + rule_id  = (known after apply)
          + type     = "REGULAR"

          + action {
              + type = "BLOCK"
            }
        }
    }

  # module.waf.aws_waf_size_constraint_set.owasp_07_size_restriction_set[0] will be created
  + resource "aws_waf_size_constraint_set" "owasp_07_size_restriction_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + size_constraints {
          + comparison_operator = "GT"
          + size                = 4096
          + text_transformation = "NONE"

          + field_to_match {
              + type = "BODY"
            }
        }
      + size_constraints {
          + comparison_operator = "GT"
          + size                = 1024
          + text_transformation = "NONE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + size_constraints {
          + comparison_operator = "GT"
          + size                = 512
          + text_transformation = "NONE"

          + field_to_match {
              + type = "URI"
            }
        }
      + size_constraints {
          + comparison_operator = "GT"
          + size                = 4093
          + text_transformation = "NONE"

          + field_to_match {
              + data = "cookie"
              + type = "HEADER"
            }
        }
    }

  # module.waf.aws_waf_size_constraint_set.owasp_08_csrf_token_size_constrain_set[0] will be created
  + resource "aws_waf_size_constraint_set" "owasp_08_csrf_token_size_constrain_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + size_constraints {
          + comparison_operator = "EQ"
          + size                = 36
          + text_transformation = "NONE"

          + field_to_match {
              + data = "x-csrf-token"
              + type = "HEADER"
            }
        }
    }

  # module.waf.aws_waf_sql_injection_match_set.owasp_01_sql_injection_set[0] will be created
  + resource "aws_waf_sql_injection_match_set" "owasp_01_sql_injection_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + sql_injection_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "BODY"
            }
        }
      + sql_injection_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + type = "BODY"
            }
        }
      + sql_injection_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + sql_injection_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + sql_injection_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + sql_injection_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + sql_injection_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + data = "Authorization"
              + type = "HEADER"
            }
        }
      + sql_injection_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + data = "Authorization"
              + type = "HEADER"
            }
        }
    }

  # module.waf.aws_waf_xss_match_set.owasp_03_xss_set[0] will be created
  + resource "aws_waf_xss_match_set" "owasp_03_xss_set" {
      + id   = (known after apply)
      + name = (known after apply)

      + xss_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "BODY"
            }
        }
      + xss_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + type = "BODY"
            }
        }
      + xss_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + xss_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + type = "QUERY_STRING"
            }
        }
      + xss_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + xss_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + type = "URI"
            }
        }
      + xss_match_tuples {
          + text_transformation = "HTML_ENTITY_DECODE"

          + field_to_match {
              + data = "cookie"
              + type = "HEADER"
            }
        }
      + xss_match_tuples {
          + text_transformation = "URL_DECODE"

          + field_to_match {
              + data = "cookie"
              + type = "HEADER"
            }
        }
    }

  # module.waf.random_id.this[0] will be created
  + resource "random_id" "this" {
      + b64         = (known after apply)
      + b64_std     = (known after apply)
      + b64_url     = (known after apply)
      + byte_length = 8
      + dec         = (known after apply)
      + hex         = (known after apply)
      + id          = (known after apply)
      + keepers     = {
          + "target_scope" = "global"
        }
    }

Plan: 21 to add, 0 to change, 0 to destroy.
```

### CODE REVIEW

**Comment by rafikurnia** [comment]
Hi @julianxhokaxhiu thanks for the PR!

From #9 I understand with the output name, it should start with letter on Terraform 0.12
But is there a problem with `random_id.this.hex` so that you propose to change it into `random_id.this.0.hex` ?

**Comment by deanillfeld** [comment]
It's required in Terraform 0.12 to always specify the index when you have a `count =` declaration on a resource. The old `random_id.this.hex` syntax is no longer valid.

**Comment by julianxhokaxhiu** [comment]
Exactly what @deanillfeld said. This is why I had to fix that one too.

**Comment by rafikurnia** [comment]
I see.. I honestly haven't explored Terraform 0.12 yet :smile: 
Thanks @deanillfeld and @julianxhokaxhiu for the explanation..

In the near future we will definitely need this module to support Terraform 0.12

I will review and do some testing to find out whether it is still possible to have one version that works on both Terraform 0.11 and 0.12 or not. I will get back to you with the updates..

**Comment by jamieneo** [comment]
I'd like to use this as well. Can we get this PR approved.

**Comment by jamie3** [comment]
Yes please approve the PR.

**Comment by jamie3** [comment]
Any ETA on when this will be approved? If not soon I'll just fork.

**Comment by julianxhokaxhiu** [comment]
@jamie3 I'm not sure. I'm still waiting for @rafikurnia feedback. On my side I cant just tell that I used and it's working just fine on top of Terraform 0.12. I hope the patch will go upstream as I don't want to keep a fork just for this patchset.

**Comment by sshvetsov** [comment]
I can understand your impatience, folks. Do understand however, this is not
a minor/patch change, but a change that breaks backwards compatibility for
existing users and requires major release.

Most of the users in our company are using Terraform 0.11 and do not
benefit from this change. Terraform 0.12 compatibility is on the roadmap,
but isn't a top priority for us at the moment.

Since this is the first time we have to deal with module losing
compatibility between Terraform versions, we are trying to figure out the
best way to update and maintain modules that require different code in
order to support different Terraform versions. If you have some concrete
ideas, please share.

At this point, we're not going to merge patches that break support for our
existing users, and we are not ready to stop support for Terraform 0.11.
Once we figure out a good way to support both 0.11 and 0.12 with minimal
overhead, we will merge this patch.

I'm glad you've found this module useful and wish to contribute. Hopefully
my explanation makes sense.

**Comment by rafikurnia** [comment]
I feel elated to know that lots of people are using this module and waiting this PR to be merged :smile: 

Also @julianxhokaxhiu, I am surprised that your patch is also working for Terraform 0.11.14, which means it can be merged without worrying about maintaining two different version.

**Comment by julianxhokaxhiu** [comment]
@rafikurnia done :) I'm happy it works on both!

File: examples/owasp-top-10/outputs.tf [hunk 1]
```diff
@@ -5,40 +5,40 @@ output "rule_group_id" {
 
 output "01_sql_injection_rule_id" {
```
Comment (rafikurnia) [inline examples/owasp-top-10/outputs.tf]: Can you help to rename this and the rest of the outputs too? The example won't work on Terraform 0.12 without changing them, isn't it..

Comment (julianxhokaxhiu) [reply examples/owasp-top-10/outputs.tf]: Oops! You're right!

Comment (julianxhokaxhiu) [reply examples/owasp-top-10/outputs.tf]: Done. Amended my previous commit.

**Comment by julianxhokaxhiu** [comment]
Can I please ask if you can also bump the version in the Terraform registry? Thank you in advance!

**Comment by rafikurnia** [comment]
Oh thanks for reminding me @julianxhokaxhiu 
Done :wink:

**Pull request merged**